mllib/src/main/scala/org/apache/spark/ml/stat/FValueTest.scala

```diff
@@ -44,7 +44,7 @@ object FValueTest {
    * @return DataFrame containing the test result for every feature against the label.
    *         This DataFrame will contain a single Row with the following fields:
    *          - `pValues: Vector`
-   *          - `degreesOfFreedom: Array[Int]`
+   *          - `degreesOfFreedom: Array[Long]`
```

Comment:
Can DoF really be so large that a long is needed? No big deal; if that's how it works now and you're just correcting docs, that's OK

Reply:
DoF = ```numSamples - 2```, so it could be very large.